test/ao/sparsity/test_sparsifier.py

```diff
@@ -55,3 +70,37 @@ def test_state_dict(self):
         assert sparsifier0.module_groups != sparsifier1.module_groups
         sparsifier1.load_state_dict(state_dict)
         assert sparsifier0.module_groups == sparsifier1.module_groups
+
+    def test_mask_squash(self):
+        model = Model()
+        sparsifier = ImplementedSparsifier(test=3)
+        sparsifier.prepare(model, [model.linear])
+        assert hasattr(model.linear, 'mask')
```

Comment:
This shouldnt be the case isn't it? Isnt mask an attr of the fake-sparse module?